sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/ProjectTypeCollectorTest.java

```diff
@@ -35,48 +35,22 @@ public void withNoProjects() {
   public void withNoFiles_skip() {
     ProjectTypeCollector underTest = new ProjectTypeCollector();
```

Comment:
I would use [sut](https://en.wikipedia.org/wiki/System_under_test) instead of `underTest`. It's short and the convention is well known.

See also the mention from Martin Fowler [here](https://martinfowler.com/articles/mocksArentStubs.html):
> I'll use System Under Test, or rather the abbreviation SUT.

Reply:
yes, I had a discussion with Pavel some time ago on this, and since I'm using more explicit names.
Let's discuss it in the "how we work" and align with everyone.